pick.lic

```diff
@@ -708,48 +714,101 @@ class Pick
 
   def loot(box_noun)
     echo "Looting #{box_noun}..." if @debug
-    if DRC.bput("open my #{box_noun}", /You open/, /^In the .* you see .*\./, 'That is already open', 'It is locked') == 'It is locked'
+    if !DRCI.open_container?(box_noun)
+    #if DRC.bput("open my #{box_noun}", /^In the .* you see .*\./, 'That is already open', 'It is locked') == 'It is locked'
```

Comment:
🧹  Can remove this commented out line since it's been replaced by DRCI.open_container?
```suggestion
```